### DISCUSSION TITLE
Does Quarkus support running GRPC on virtual threads?

### DISCUSSION THREAD

**Discussion by sixwaaaay** [quarkusio/quarkus#32327] [Q&A] [answered]
I noticed that Quarkus provides [documentation ](https://quarkus.io/guides/virtual-threads)and support for virtual threads in HTTP handlers, but I haven't been able to find any information on gRPC. Does Quarkus currently support running gRPC on virtual threads?

[a comment by an automated account is not shown]

**Reply by cescoffier** [reply]
No, not yet.

**Comment by cescoffier** [comment, marked as answer]
Let me add more details. We needed to wait for the new server and client so we have more control on how calls are handled. With the original one we would have pinned the carrier thread. 

So, we can now make progress on the subject.
